rbac/api/cross_access/view.py

```diff
@@ -107,6 +116,54 @@ def list(self, request, *args, **kwargs):
             return self.replace_user_id_with_info(result)
         return result
 
+    def partial_update(self, request, *args, **kwargs):
```

Comment:
Should we do anything if you don't provide any update data at all?

Reply:
I *think* the serializer update() method can handle this now, right? If nothing comes in, that's fine, nothing happens and we return the same object?